src/ufixed.jl

```diff
@@ -10,18 +10,34 @@ end
 
   rawtype{T,f}(::Type{UFixed{T,f}}) = T
 nbitsfrac{T,f}(::Type{UFixed{T,f}}) = f
+typechar{X<:UFixed}(::Type{X}) = 'N'
+signbits{X<:UFixed}(::Type{X}) = 0
+
+for T in (UInt8, UInt16, UInt32, UInt64)
+    for f in 0:sizeof(T)*8
+        sym = Symbol(takebuf_string(showtype(_iotypealias, UFixed{T,f})))
+        @eval begin
+            typealias $sym UFixed{$T,$f}
+            export $sym
+        end
+    end
+end
 
+# ASCII typealiases
+typealias U8       UFixed{UInt8,8}
 typealias UFixed8  UFixed{UInt8,8}
 typealias UFixed10 UFixed{UInt16,10}
 typealias UFixed12 UFixed{UInt16,12}
 typealias UFixed14 UFixed{UInt16,14}
 typealias UFixed16 UFixed{UInt16,16}
+typealias U16      UFixed{UInt16,16}
```

Comment:
The U8 notation is pretty intuitive for someone coming from an image processing background since most packages use the general datatypes defined in their languages as the base Image type unlike Images.jl where we have FixedPointNumbers to abstract that out and give us one less thing to worry about :smile: If we are going to remove U8 and friends, then we need to support it with good documentation in the Images.jl packages (explaining how FixedPointNumbers works, why it helps and the notation). This does seem like a big change though so if you could also have a big warning message when you deprecate U8 (something that explains how to use the Nxfy notation too) it will really help users. Apart from this, I like the new implementation and even if there is a bit of a learning curve to it, I guess its worth it in the long run :smile: